modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -515,6 +542,21 @@ public override void _EnablePlugin()
                 ["hint_string"] = settingsHintStr
             });
 
+            _editorSettings.AddPropertyInfo(new Godot.Collections.Dictionary
+            {
+                ["type"] = (int)Variant.Type.String,
+                ["name"] = Settings.ExecPath,
+                ["hint"] = (int)PropertyHint.LocalizableString,
+                ["hint_string"] = $"exec command"
```

Comment:
```suggestion
                ["hint"] = (int)PropertyHint.File
```

`LocalizableString` means that the property is a localizeable string, not that the hint should be localized.